BunqSdk.Samples/PaymentListSample.cs

```diff
@@ -24,32 +24,48 @@ public class PaymentListSample : ISample
         /// <summary>
         /// Constants to be changed to run the example.
         /// </summary>
-        private const int USER_ITEM_ID = 0; // Put your user ID here
-        private const int MONETARY_ACCOUNT_ITEM_ID = 0; // Put your monetary account ID here
-
+        private const int USER_ITEM_ID = 0; // Put your user ID here
+
         public void Run()
         {
             var apiContext = ApiContext.Restore();
             var paginationCountOnly = new Pagination
             {
                 Count = PAGE_SIZE,
             };
-            Console.WriteLine(MESSAGE_LATEST_PAGE_IDS);
-            var paymentResponse = Payment.List(apiContext, USER_ITEM_ID, MONETARY_ACCOUNT_ITEM_ID,
-                paginationCountOnly.UrlParamsCountOnly);
-            PrintPayments(paymentResponse.Value);
-            var pagination = paymentResponse.Pagination;
 
-            if (pagination.HasPreviousPage())
-            {
-                Console.WriteLine(MESSAGE_SECOND_LATEST_PAGE_IDS);
-                var previousPaymentResponse = Payment.List(apiContext, USER_ITEM_ID, MONETARY_ACCOUNT_ITEM_ID,
-                    pagination.UrlParamsPreviousPage);
-                PrintPayments(previousPaymentResponse.Value);
-            }
-            else
+            var monetaryAccounts = MonetaryAccount.List(apiContext, USER_ITEM_ID).Value;
+
+            foreach (var monetaryAccount in monetaryAccounts)
             {
-                Console.WriteLine(MESSAGE_NO_PRIOR_PAYMENTS_FOUND);
+                var acctBank = monetaryAccount.MonetaryAccountBank;
+
+                if (!acctBank.Id.HasValue)
```

Comment:
I believe this is not `really` necessary.
I think I know why you made the `id` nullable. it's derived from the `active record`-ish design of the SDK API and the fact the DTO is combined with the actions (requests / behavioral aspects). Most likely the `id` is never going to be null for existing accounts - but this is my assumption. Therefore I put a guard just in case.